src/STM32Ethernet.cpp

```diff
@@ -73,6 +73,15 @@ int EthernetClass::maintain(){
   return rc;
 }
 
+/*
+ * This function updates the LwIP stack and can be called to be sure to update
+ * the stack (e.g. in case of a long loop).
+ */
+void EthernetClass::update(void)
```

Comment:
why do you need this new update interface ? as far as I understood the list of callbacks is a table in core ...  so stm32_eth_scheduler will be called directly  .... isn't it ?

Reply:
Right @LMESTM .
I think @fprwi6labs add it to enhance EthernetClass API that user can use in his sketch.

Reply:
how to use update is not clear to me - please add more information in the commit message why we make the change and how this will/can be used.

Reply:
It is just in case you go in a blocking loop inside the loop() function, neither updateCoreCallback nor stm32_eth_scheduler will be called. So I just give a possibility to the user to update the LwIP stack to keep Ethernet alive.

Reply:
Ok - I'd like to see this kind of information in the commit message as well.
Typically I guess that you don't want both APIs to be called, user should select one of them ? What is the timing of a loop that requires this API to be called from user application ? 
Other point :  is update a standard name, or could be consider EthernetClass::schedule() ?

Reply:
User can use both method. 
I don't master enough the LwIP stack to give you a figure.
There isn't a standard name so I can change.